trestle/tasks/xlsx_to_oscal_component_definition.py

```diff
@@ -0,0 +1,671 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from mypyc.primitives import str_ops
```

Comment:
Was this intended? I think you want to push this lower?

Reply:
No, was probably eclipse giving unwanted help. Fixed.

Reply:
- nist  catalog copy removed.
- could you say which spread sheet are corrupt and which are not, if any.  don't understand why that would be.
- working on streamlining main execute function...
- could you expand on the "vendor specific" concern?
- any particular reason for preference of pandas over openpyxl? the latter seems pretty good.

Reply:
Still using openpyxl, which seems to work very nicely.

Reply:
Great - I see no issues as pandas effectively wraps openpyxl anyway.